app/templates/pwa/ts/register-service-worker.ts

```diff
@@ -0,0 +1,42 @@
+import { register } from 'register-service-worker';
+
+// The ready(), registered(), cached(), updatefound() and updated()
+// events passes a ServiceWorkerRegistration instance in their arguments.
+// ServiceWorkerRegistration: https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+register(process.env.SERVICE_WORKER_FILE!, {
```

Comment:
```suggestion
register(process.env.SERVICE_WORKER_FILE, {
```

We might consider some alternative solutions:

1. Creating another `env.d.ts` file in `src-pwa` directory, with the following content:
```ts
declare namespace NodeJS {
  interface ProcessEnv {
    SERVICE_WORKER_FILE: string
  }
}
```
2. Adding `SERVICE_WORKER_FILE: string` entry to the main `env.d.ts` file in `src` folder.

Reply:
Being just one, I took the lazy route. But you're right, I added the env file, which is the correct way